dev/build/windows/makecoq_mingw.sh

```diff
@@ -1320,6 +1326,22 @@ function make_coq_installer {
   fi
 }
 
+###################### ADDONS #####################
+
+function make_addon_bignums {
+  if build_prep https://github.com/coq/bignums/archive/ master zip 1 bignums-8.8.0; then
+    logn make make all
+    logn make-install make install
+    build_post
+  fi
+}
+
+function make_addons {
+  if [ ! -z "$COQ_ADDONS" ]; then
+    for addon in $COQ_ADDONS; do make_addon_$addon; done
```

Comment:
Why test for nonzeroness? A loop over nothing does nothing.

Reply:
oops, thanks